docs/Lama2/docs/tutorials/examples.md

```diff
@@ -69,8 +69,18 @@ Get [Source File](https://github.com/HexmosTech/Lama2/tree/main/examples/0003_co
 
 ## Environment Variables: Switch base URL
 
```

Comment:
### Case 1: `l2.env` adjacent to an API file

For any given `.l2` file, one can place an `l2.env` file to store relevant variables. These variables will be available to be used within the API file

[example]

### Case 2: Project-level variables

In Lama2, you can have a large number of API files stored in a hierarchical folder configuration. The root of such a project can be signified through `l2config.env`:

[API Hub folder screenshot with `l2config.env` highlighted]

Within such a structure, you can have an API file anywhere, which can use variables defined in the project-level variables:

[Example l2config.env and api.l2 with appropriate folder structure; the example must be hosted in Lama2 repo (see other examples)]

### Case 3: Override Project-level variable with local variable

....

Reply:
Got it